File: singleview/singleview/ViewController.m
//
//  ViewController.m
//  singleview
//
//  Created by Atlas Huang (RD-TW) on 2019/3/12.
//  Copyright © 2019 Atlas Huang (RD-TW). All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController
@synthesize labelEnd;

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    UILabel *label = [[UILabel alloc] initWithFrame:CGRectMake(200, 200, 200, 200)];
    label.text = @"Produced by Atlas";
    [self.view addSubview:label];
    
    labelEnd.text = @"This is a test string for End word";
    
    print(@"Atlas");
    aaa();
}

void (^print)(NSString *) = ^(NSString* name) {
    NSLog(@"Hello world %@", name);
};

void (^aaa)() = ^(){
    NSLog(@"aaa");
};

@end

File: helloworld/helloworld/ViewController.swift
//
//  ViewController.swift
//  helloworld
//
//  Created by Atlas Huang (RD-TW) on 2019/3/18.
//  Copyright © 2019 Atlas Huang (RD-TW). All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    @IBAction func showMessage() {
        let alertController = UIAlertController(title: "Welcome to my app", message: "Hello world", preferredStyle: .alert)
        alertController.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
        present(alertController, animated: true, completion: nil)        
    }


}

